docs/modules/index.md

```diff
@@ -205,8 +205,6 @@ In order to simplify the creation of the container for a given module, `Testcont
 - `testcontainers.WithConfigModifier`: a function that sets the config Docker type for the container request. Please see [Advanced Settings](../features/creating_container.md#advanced-settings) for more information.
 - `testcontainers.WithEndpointSettingsModifier`: a function that sets the endpoint settings Docker type for the container request. Please see [Advanced Settings](../features/creating_container.md#advanced-settings) for more information.
 - `testcontainers.WithHostConfigModifier`: a function that sets the host config Docker type for the container request. Please see [Advanced Settings](../features/creating_container.md#advanced-settings) for more information.
-- `testcontainers.WithWaitStrategy`: a function that sets the wait strategy for the container request, adding all the passed wait strategies to the container request, using a `testcontainers.MultiStrategy` with 60 seconds of deadline. Please see [Wait strategies](../features/wait/multi.md) for more information.
```

Comment:
See https://github.com/testcontainers/testcontainers-go/pull/2989/files#diff-bf2f2ea9f93d17fd1a016e2893c5b7fe7dae04263b1662c8e03926e4e728f495R199-R200 for the duplicated options